Assets/MRTK/Core/Interfaces/InputSystem/IMixedRealityTeleportPointer.cs

```diff
@@ -21,5 +21,10 @@ public interface IMixedRealityTeleportPointer : IMixedRealityPointer
         /// The Y orientation of the pointer - used for touchpad rotation and navigation
         /// </summary>
         float PointerOrientation { get; }
+
+        /// <summary>
+        /// Teleport pointer will only respond to input events for teleportation that match this MixedRealityInputAction.
+        /// </summary>
+        MixedRealityInputAction TeleportAction { get; set; }
```

Comment:
Removing the process of "ask the pointer for its action" would also remove this breaking change.